tests/tsdk/store.go

```diff
@@ -0,0 +1,99 @@
+package tsdk
+
+import (
+	"io"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/store"
+	types "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/tendermint/tendermint/libs/log"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+	dbm "github.com/tendermint/tm-db"
+)
+
+// NewCommitMultiStore creats SDK Multistore
```

Comment:
Is there a way to document (or link to docs) the purpose of tkeys and keys for those unfamiliar with the SDK?

Reply:
the type already provides that. Also, we are having it in every module we are implementing.